src/EnergyPlus/SizingManager.cc

```diff
@@ -3373,6 +3360,32 @@ void GetZoneSizingInput(EnergyPlusData &state)
     }
 }
 
+void ReportTemperatureInputError(
```

Comment:
At first I wasn't quite sure why you were passing a zero in here, but this makes good sense now.  Nice reuse of the new worker function.